common/scripts/configs/system_configs.sql

```diff
@@ -84,63 +74,51 @@ do $$
     end if;
 
     if not exists (select 1 from information_schema.columns where table_name = 'systemConfigs' and column_name = 'customHostDockerVersion') then
-      alter table "systemConfigs" add column "customHostDockerVersion" VARCHAR(24);
-    end if;
-
-    if not exists (select 1 from information_schema.columns where table_name = 'systemConfigs' and column_name = 'wwwPort') then
-      alter table "systemConfigs" add column "wwwPort" INTEGER;
-    end if;
-
-    if not exists (select 1 from information_schema.columns where table_name = 'systemConfigs' and column_name = 'redisUrl') then
-      alter table "systemConfigs" add column "redisUrl" varchar(255);
+      alter table "systemConfigs" add column "customHostDockerVersion" VARCHAR(24) default '1.9.1';
```

Comment:
@ric03uec  - the current docker default in base's state.json.example is 1.9.1. What do we want the default to be in Admiral? Keep it 1.9.1, make it 1.13, or...???